savant/utils/logging/savant_rs_handler.py

```diff
@@ -0,0 +1,38 @@
+"""SavantRsLoggingHandler module."""
+import logging
+import pretty_traceback
+from savant_rs.logging import log
+from .log_utils import log_level_py_to_rs
+
+
+class SavantRsLoggingHandler(logging.Handler):
+    """Custom logging Handler that passes the log messages
+    to the rust log from savant_rs with appropriate log level.
+
+    Hardcoded to use two formatters (no need to specify during logging configuration):
+    1. basic formatter for all messages under ERROR priority
+    2. pretty_traceback formatter for ERROR and CRITICAL messages
+    """
+
+    def __init__(self) -> None:
+        logging.Handler.__init__(self)
+        self.pretty_trace_back_threshold = logging.ERROR
+        self.formatter_basic = logging.Formatter()
```

Comment:
Add timestamps to log formatter
```
DEBUG savant::pipeline::demo                                 > Requesting new sink pad on muxer: sink_0
INFO  savant::pipeline::demo                                 > Added source cam-1
```

Reply:
The idea is that the underlying rust logger (currently [pretty_env_logger](https://github.com/insight-platform/savant-rs/blob/09c5533740de5267912df6fba3556dfa1a0219cf/savant_python/src/lib.rs#L65)) will do the formatting.

Reply:
@abramov-oleg you are right